crypto/rsa_extra/rsa_test.cc

```diff
@@ -639,6 +639,7 @@ TEST(RSATest, BadExponent) {
   ERR_clear_error();
 }
 
+#if !defined(BORINGSSL_FIPS)
```

Comment:
We could think about using the `AWSLC_FIPS` macro we have? 
That might diverge from the FIPS ifdef in the `crypto/internal.h` file we're  changing in this PR though. But then again, the separate macros could also help keep track of which ifdefs AWS-LC created.

Reply:
Done in PR#326